README.md

```diff
@@ -88,7 +88,7 @@ TypeScript will be added as a local dependency to the project, so no need to ins
 yarn
 ```
 
-#### Mac M1 chip users will need to prepend `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`.
+#### Mac M1, M2 chip users will need to prepend `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`.
```

Comment:
Maybe use "and" instead of ","?

Reply:
Could potentially future-proof it a bit by saying "Mac users with M-series chips..." since there will probably be an M3

Reply:
Done... Thanks for the suggestion.